Library/Formula/bazaar.rb

```diff
@@ -27,5 +28,23 @@ def install
     libexec.install 'bzr', 'bzrlib'
 
     bin.install_symlink libexec+'bzr'
+
+    # install the plugins under /var/bazaar/plugins/
+    plugins_orig = libexec/"bzrlib/plugins"
+    plugins_new = var/"bazaar/plugins"
+
+    Dir[plugins_orig/"*"].each do |plugin|
+      path = Pathname.new plugin
+      plugins_new.install plugin unless File.exist? (plugins_new/path.basename)
+    end
+
+    rm_rf plugins_orig
+    ln_s plugins_new, plugins_orig
+  end
```

Comment:
@mikemcquaid I’ve never used `brew bottle` before and it’s not documented in the man page.

I reinstalled Bazaar with `--build-bottle`, and then:

```
$ brew bottle bazaar
==> Determining bazaar bottle revision...
==> Bottling bazaar-2.6.0_1.yosemite.bottle.tar.gz...
==> Detecting if bazaar-2.6.0_1.yosemite.bottle.tar.gz is relocatable...
./bazaar-2.6.0_1.yosemite.bottle.tar.gz
  bottle do
    sha1 "6a3eb3f77852509b42807ae7e9fa8c615d73aa76" => :yosemite
  end
```